### ISSUE TITLE
Inserttag for each category

### ISSUE THREAD

**Issue by jenda77** [codefog/contao-news_categories#73]
It could be very helpful to have the possibility to set an inserttag for each category to link to that. By now it is necessary to copy the link to the category manually. I think it could be very useful when the page name will be changed then it is not necessary to change the link to the category page manually again. See: http://neu.berlin-pm.com/en/our-partners.html 

What do you think?

**Comment by fritzmg** [comment]
:+1:

**Comment by jenda77** [comment]
Perfect. And of course it should support the language variables and the multilanguage extension like https://github.com/verstaerker/i18nl10n Thank you. :+1:

**Comment by jenda77** [comment]
I would like to ask you when you want to release it. Or is there a possibility to do it on my own now?

**Comment by qzminski** [comment]
I think the next major release will be either in mid December or January.

**Comment by jenda77** [comment]
Thank you. And there is no way to do it on my own, right?

**Comment by qzminski** [comment]
You can update the code on your installation manually of course.

**Comment by jenda77** [comment]
Sure, but I have no idea how the code should be. :-) Something like this? https://docs.contao.org/books/cookbook/de/Eigene-Inserttags.html

**Comment by fritzmg** [comment]
You could put the following code
```php
<?php

$strParam = \NewsCategories\NewsCategories::getParameterName();
$strClass = \NewsCategories\NewsCategories::getModelClass();
$objCategory = $strClass::findPublishedByIdOrAlias( \Input::get( $strParam ) );

if( $objCategory !== null )
{
	if( $objCategory->jumpTo )
	{
	    $objJump = \PageModel::findByPk( $item['jumpTo'] );
	    if( $objJump !== null )
	    {
	        echo $this->generateFrontendUrl( $objJump->row() );	
	    }	
	}
}
```
into `/templates/category.php` and then use
```
{{file::category.php?category=foo}}
```
This will echo the redirect page of the category, if set.

**Comment by jenda77** [comment]
Thank you, Dankeschön. But before have I to set the link in each category? I would like to have a inserttag for the main navigation to set the link to each category in each page. Otherwise I thought I could use the nav_categories template and change it for the main navigation. But it could be a little bit complicated way because I have to combinate two navigation templates - the default and the one for the categories. 

The best way could be to be able to choose the news category directly in the page settings in the page tree and use them as link.

**Comment by fritzmg** [comment]
> But before have I to set the link in each category? I would like to have a inserttag for the main navigation to set the link to each category in each page. Otherwise I thought I could use the nav_categories template and change it for the main navigation. But it could be a little bit complicated way because I have to combinate two navigation templates - the default and the one for the categories.

Hm, not sure if I understand you correctly. But if you want to use the feature of setting a redirect page for each category and want to use that also in the regular category navigation, you can use this gist: https://gist.github.com/fritzmg/81266c77526262fbbf0a

**Comment by jenda77** [comment]
Thank you but, no, I need it for the main navigation.

**Comment by jenda77** [comment]
I have combinated it with the main navigation. The navigation items are there but the links are not the correct ones - I have used the code from https://gist.github.com/fritzmg/81266c77526262fbbf0a to set the redirect page. But the links are wrong to compare with the categories navigation on the same page. See http://babymanufactur.ppam.de/produkte/produktkategorie.html and the main navigation under "Produkte".

**Comment by fritzmg** [comment]
You don't need a special insert tag for that, you just have to insert the regular news categories navigation module for this particular main navigation item. There are many different ways to do that, my preferred way is to extend the DCA of `tl_page` with a _Custom submenu items_ text field where I can simply insert an insert tag, for instance `{{insert_module::*}}`. And in the regular `nav_default` template I exchange
```php
<?= $item['subitems'] ?>
```
with
```php
<?= $item['customsubitems'] ?: $item['subitems'] ?>
```

**Comment by jenda77** [comment]
Thank you, this is good way too. But how do I get the correct link to the category with the related news? Something like this http://babymanufactur.ppam.de/produkte/produktkategorie/kategorie/wickelkommode.html

**Comment by fritzmg** [comment]
Ah, so you want to also display the news items for each category as a further sub menu? Well you could create a news list module on the fly, with the correct categories set in the model.

**Comment by jenda77** [comment]
No, no news items in the submenu. The news items should be displayed according to the chosen category. 

I think my dca code for the own field in the page settings is not correct. Could you look at it, please? 

```
<?php 
 
//Anzeige im Backend
$GLOBALS['TL_DCA']['tl_page']['palettes']['default'] = str_replace("description", "description,customsubitems", $GLOBALS['TL_DCA']['tl_page']['palettes']['default']);
 
//Beschreibung des Feldes
$GLOBALS['TL_DCA']['tl_page']['fields']['customsubitems'] = array
(
	'label'                   => &$GLOBALS['TL_LANG']['tl_page']['customsubitems'],
	'inputType'               => 'text',
	'eval'                    => array('maxlength'=>255, 'tl_class'=>'clr long'),
	'sql'                     => "varchar(255) NOT NULL default ''"
);

```

**Comment by fritzmg** [comment]
> No, no news items in the submenu. The news items should be displayed according to the chosen category.

I still don't understand. That means you just need a regular news list module? The news list module already displays the news items according to the chosen category, if you activated the option.

> I think my dca code for the own field in the page settings is not correct. Could you look at it, please?

Looks fine. Where did you insert it? If you put that into the dcaconfig.php you need to be careful with the `str_replace`, the dcaconfig.php might be loaded more than once.

**Comment by jenda77** [comment]
Ok, lieber auf Deutsch, Ich möchte, wenn man die Kategorie in der Hauptnavigation anklickt, dass in der Newsliste die der ausgewählten Kategorie zugehörige Beiträge stehen. Das heißt, es sollte genauso funktionieren wie das Modul nav_categories, das sich schon auf der Beispielsseite befindet.

**Comment by fritzmg** [comment]
> Ich möchte, wenn man die Kategorie in der Hauptnavigation anklickt, dass in der Newsliste die der ausgewählten Kategorie zugehörige Beiträge stehen.

Das ist ja ein Standardfeature der Newslist von news_categories, du musst nur die Option "Nach Kategorien filtern" aktivieren.

> Das heißt, es sollte genauso funktionieren wie das Modul nav_categories, das sich schon auf der Beispielsseite befindet.

Verstehe ich nicht, das sind doch grund verschiedene Dinge ;)

**Comment by jenda77** [comment]
Das ist alles aktiviert. Es geht darum, dass es die richtigen Links in der Navigation vorkommen, wie es im Modul nav_categories der Fall ist. Sonst werden doch keine zugehörigen News angezeigt.

**Comment by fritzmg** [comment]
In der Navigation hast du doch schon die korrekten Links, wenn du so vorgegangen bist, wie oben beschrieben. Du fügst ja das Modul _Liste mit Nachrichten-Kategeorien_ in die Navigation ein. Diese Navigation hat dann die korrekten Links zu den jeweiligen Nachrichtenkategorien auf der Zielseite.

**Comment by jenda77** [comment]
Deinen Code habe ich in die Template eingetragen. Das hilft mir nicht. Die List der zugehörigen Sachen wird doch nach einer anderen Login erstellt.

**Comment by fritzmg** [comment]
> Die List der zugehörigen Sachen wird doch nach einer anderen Login erstellt.

Was meinst du damit?

Sorry, aber ich glaube ich verstehe immer noch nicht was du vorhast ;)

**Comment by jenda77** [comment]
Es ist sehr einfach. Die Unterpunkte der Hauptnavigation sollen die selben Links haben wie das Modul nav_categories, damit die Liste mit den zugehörigen News der ausgewählten Kategorie erstellt wird. Darum geht es mir die ganze Zeit.

**Comment by fritzmg** [comment]
Ja, das haben sie ja bereits auf deiner Website? Und genau das wäre ja der Fall, wenn du das Nachrichten Kategorie Navigations Modul (_Liste mit Nachrichten-Kategeorien_) eben so in die Navigation einfügst, wie beschrieben.

**Comment by fritzmg** [comment]
Soweit ich sehen kann hast du zB folgende Seitenstruktur:

* Home
* Baby Manfucatur
* Service
* Aktuelles
* Marken
* Produkte
* Kontakt

Und im Level 2 der Seite __Produkte__ hast du offensichtlich schon das Modul _Liste mit Nachrichten-Kategeorien_ eingefügt, welches dir die Produkt-Kategorien ausgibt. In diesem Modul hast du eingestellt, dass die Zielseite die Seite mit dem Alias `produkte/produktkategorie` ist.

Das Ergebnis ist dann folgende HTML Struktur in deinem Menü:
```html
…
<li class="trail menu-item">
	<a href="produkte.html" title="Produkte" class="trail" rel="nofollow" role="menuitem">Produkte</a>
	<ul class="level_2">
		<li class="news_category_102 first even submenu">
			<a href="produkte/produktkategorie/kategorie/kinder.html" class="news_category_102 first even submenu" title="Kinder">Kinder</a>
			<ul class="level_2">
				<li class="news_category_103 first even submenu">
					<a href="produkte/produktkategorie/kategorie/kinderzimmer.html" class="news_category_103 first even submenu" title="Kinderzimmer">Kinderzimmer</a>
					<ul class="level_2">
						<li class="news_category_106 first even">
							<a href="produkte/produktkategorie/kategorie/kinderbetten.html" class="news_category_106 first even" title="Kinderbetten">Kinderbetten</a>
						</li>
						<li class="news_category_104 last odd">
							<a href="produkte/produktkategorie/kategorie/kleiderschraenke.html" class="news_category_104 last odd" title="Kleiderschränke">Kleiderschränke</a>
						</li>
					</ul>
				</li>
				<li class="news_category_116 last odd">
					<a href="produkte/produktkategorie/kategorie/kinderaccessoires.html" class="news_category_116 last odd" title="Kinderaccessoires">Kinderaccessoires</a>
				</li>
			</ul>
		</li>
		<li class="news_category_109 odd submenu trail">
			<a href="produkte/produktkategorie/kategorie/spielen-und-schenken.html" class="news_category_109 odd submenu trail" title="Spielen &amp; Schenken">Spielen & Schenken</a>
			<ul class="level_2">
				<li class="news_category_105 first even active">
					<span class="news_category_105 first even active">Kinderspielzeug</span>
				</li>
				<li class="news_category_110 last odd">
					<a href="produkte/produktkategorie/kategorie/babyspielzeug.html" class="news_category_110 last odd" title="Babyspielzeug">Babyspielzeug</a>
				</li>
			</ul>
		</li>
		<li class="news_category_100 last even submenu">
			<a href="produkte/produktkategorie/kategorie/baby.html" class="news_category_100 last even submenu" title="Baby">Baby</a>
			<ul class="level_2">
				<li class="news_category_101 first last even submenu">
					<a href="produkte/produktkategorie/kategorie/babyzimmer.html" class="news_category_101 first last even submenu" title="Babyzimmer">Babyzimmer</a>
					<ul class="level_2">
						<li class="news_category_118 first even">
							<a href="produkte/produktkategorie/kategorie/laufstall.html" class="news_category_118 first even" title="Laufstall">Laufstall</a>
						</li>
						<li class="news_category_99 odd">
							<a href="produkte/produktkategorie/kategorie/wickelkommode.html" class="news_category_99 odd" title="Wickelkommoden">Wickelkommoden</a>
						</li>
						<li class="news_category_98 last even">
							<a href="produkte/produktkategorie/kategorie/babybett.html" class="news_category_98 last even" title="Babybetten">Babybetten</a>
						</li>
					</ul>
				</li>
			</ul>
		</li>
	</ul>
</li>
…
```
In der Hauptnavigation bei der regulären Seite "Produkte" befindet sich als als Untermenü somit die Ausgabe der Nachrichten Kategorien.

**Comment by jenda77** [comment]
Ja, das ist richtig und ich weiß, was ich richtig umgesetzt habe. ;-) Aber dadurch funktioniert die Auflistung immer noch nicht. Das ist doch offensichtlich. Bitte die Funktion von dem Modul nav_newscategories mit der eingeschränkten Verlinkung in der Hauptnavigation vergleichen. Es geht mir um die Filterfuntion des Moduls nav_newscategories, d.h., durch die Links werden die News gefiltert und genau diese Links möchte ich auch in der Hauptnavigation haben. Nichts mehr, nichts weniger. ;-) 

26. 11. 2015 v 18:01, fritzmg <notifications@github.com>:

Soweit ich sehen kann hast du zB folgende Seitenstruktur:

Home
Baby Manfucatur
Service
Aktuelles
Marken
Produkte
Kontakt
Und im Level 2 der Seite Produkte hast du offensichtlich schon das Modul Liste mit Nachrichten-Kategeorien eingefügt, welches dir die Produkt-Kategorien ausgibt. In diesem Modul hast du eingestellt, dass die Zielseite die Seite mit dem Alias produkte/produktkategorie ist.

Das Ergebnis ist dann folgende HTML Struktur in deinem Menü:

…
<li class="trail menu-item">
    <a href="produkte.html" title="Produkte" class="trail" rel="nofollow" role="menuitem">Produkte</a>
    <ul class="level_2">
        <li class="news_category_102 first even submenu">
            <a href="produkte/produktkategorie/kategorie/kinder.html" class="news_category_102 first even submenu" title="Kinder">Kinder</a>
            <ul class="level_2">
                <li class="news_category_103 first even submenu">
                    <a href="produkte/produktkategorie/kategorie/kinderzimmer.html" class="news_category_103 first even submenu" title="Kinderzimmer">Kinderzimmer</a>
                    <ul class="level_2">
                        <li class="news_category_106 first even">
                            <a href="produkte/produktkategorie/kategorie/kinderbetten.html" class="news_category_106 first even" title="Kinderbetten">Kinderbetten</a>
                        </li>
                        <li class="news_category_104 last odd">
                            <a href="produkte/produktkategorie/kategorie/kleiderschraenke.html" class="news_category_104 last odd" title="Kleiderschränke">Kleiderschränke</a>
                        </li>
                    </ul>
                </li>
                <li class="news_category_116 last odd">
                    <a href="produkte/produktkategorie/kategorie/kinderaccessoires.html" class="news_category_116 last odd" title="Kinderaccessoires">Kinderaccessoires</a>
                </li>
            </ul>
        </li>
        <li class="news_category_109 odd submenu trail">
            <a href="produkte/produktkategorie/kategorie/spielen-und-schenken.html" class="news_category_109 odd submenu trail" title="Spielen &amp; Schenken">Spielen & Schenken</a>
            <ul class="level_2">
                <li class="news_category_105 first even active">
                    <span class="news_category_105 first even active">Kinderspielzeug</span>
                </li>
                <li class="news_category_110 last odd">
                    <a href="produkte/produktkategorie/kategorie/babyspielzeug.html" class="news_category_110 last odd" title="Babyspielzeug">Babyspielzeug</a>
                </li>
            </ul>
        </li>
        <li class="news_category_100 last even submenu">
            <a href="produkte/produktkategorie/kategorie/baby.html" class="news_category_100 last even submenu" title="Baby">Baby</a>
            <ul class="level_2">
                <li class="news_category_101 first last even submenu">
                    <a href="produkte/produktkategorie/kategorie/babyzimmer.html" class="news_category_101 first last even submenu" title="Babyzimmer">Babyzimmer</a>
                    <ul class="level_2">
                        <li class="news_category_118 first even">
                            <a href="produkte/produktkategorie/kategorie/laufstall.html" class="news_category_118 first even" title="Laufstall">Laufstall</a>
                        </li>
                        <li class="news_category_99 odd">
                            <a href="produkte/produktkategorie/kategorie/wickelkommode.html" class="news_category_99 odd" title="Wickelkommoden">Wickelkommoden</a>
                        </li>
                        <li class="news_category_98 last even">
                            <a href="produkte/produktkategorie/kategorie/babybett.html" class="news_category_98 last even" title="Babybetten">Babybetten</a>
                        </li>
                    </ul>
                </li>
            </ul>
        </li>
    </ul>
</li>
…
In der Hauptnavigation bei der regulären Seite "Produkte" befindet sich als als Untermenü somit die Ausgabe der Nachrichten Kategorien.

**Comment by fritzmg** [comment]
Was funktioniert an der Auflistung nicht? Das was du schon hast ist ja schon exakt das was du willst. Du hast in der Hauptnavigation das nav_newscategories Modul eingebunden.

**Comment by jenda77** [comment]
Danke für deine Mühe und Geduld. :-) Leider funktioniert die Erstellung der Links nicht immer richtig. Es wird nicht das Alias der Seite, die das Modul als Submenu beinhaltet, übernommen, dadurch entsteht so ein Link http://babymanufactur.ppam.de/index/kategorie/spielen-und-schenken.html. D.h., statt „index“ sollte da das Alias der Seite mit dem Submenu stehen. 

Kann es sein, dass mein Modul erst später geladen werden muss? Dieses heißt tl_page_submenu. Oder woran kann es liegen, an der nav_newscategorie vermutlich?

nav_default

```

<ul class="<?= $this->level ?> <?php if ($this->level == 'level_1'): ?>menu<?php endif; ?>" <?php if ($this->level == 'level_1'): ?>id="menu-main-menu"<?php endif; ?> role="<?= ($this->level == 'level_1') ? 'menubar' : 'menu' ?>">
  <?php foreach ($this->items as $item): ?>
    <?php if ($item['isActive']): ?>
      <li class="<?= $item['class'] ?> menu-item current-menu-item"><span class="<?= $item['class'] ?>" role="menuitem"<?php if (!empty($item['subitems'])): ?> aria-haspopup="true"<?php endif; ?>><?= $item['link'] ?></span><?= $item['customsubitems'] ?: $item['subitems'] ?></li>
<?php else: ?>
      <li<?php if ($item['class']): ?> class="<?= $item['class'] ?> menu-item <?php if ($item['customsubitems']): ?>submenu<?php endif; ?>"<?php endif; ?>><a href="<?= $item['href'] ?>" title="<?= $item['pageTitle'] ?: $item['title'] ?>"<?php if ($item['class']): ?> class="<?= $item['class'] ?>"<?php endif; ?><?php if ($item['accesskey'] != ''): ?> accesskey="<?= $item['accesskey'] ?>"<?php endif; ?><?php if ($item['tabindex']): ?> tabindex="<?= $item['tabindex'] ?>"<?php endif; ?><?php if ($item['nofollow']): ?> rel="nofollow"<?php endif; ?><?= $item['target'] ?> role="menuitem"<?php if (!empty($item['subitems'])): ?> aria-haspopup="true"<?php endif; ?>><?= $item['link'] ?></a><?= $item['customsubitems'] ?: $item['subitems'] ?></li>
    
    <?php endif; ?>
  <?php endforeach; ?>
</ul>
```

nav_newscategories für die Hauptnavigation

```
<ul class="level_2">
    <?php foreach ($this->items as $item): ?>
    <?php

// build link to jumpTo page
if( $item['jumpTo'] )
{
    $objJump = \PageModel::findByPk( $item['jumpTo'] );
    if( $objJump !== null )
        $item['href'] = $this->generateFrontendUrl( $objJump->row() );
}

    ?>
    <?php if ($item['isActive']): ?>
    <li class="<?php echo $item['class']; ?>"><span class="<?php echo $item['class']; ?>"><?php echo $item['link']; ?></span><?php if ($this->showQuantity): ?> <span class="quantity">(<?php echo $item['quantity']; ?>)</span><?php endif; ?><?php echo $item['subitems']; ?></li>
    <?php else: ?>
    <li class="<?php echo $item['class']; ?>"><a href="<?php echo $item['href']; ?>" class="<?php echo $item['class']; ?>" title="<?php echo $item['linkTitle']; ?>"><?php echo $item['link']; ?><?php if ($this->showQuantity): ?> <span class="quantity">(<?php echo $item['quantity']; ?>)</span><?php endif; ?></a><?php echo $item['subitems']; ?></li>
    <?php endif; ?>
    <?php endforeach; ?>
</ul>

```

Liebe Grüße,

Jan

**Comment by jenda77** [comment]
Danke für deine Mühe und Geduld. :-) Leider funktioniert die Erstellung der Links nicht immer richtig. Es wird nicht das Alias der Seite, die das Modul als Submenu beinhaltet, übernommen, dadurch entsteht so ein Link http://babymanufactur.ppam.de/index/kategorie/spielen-und-schenken.html. D.h., statt „index“ sollte da das Alias der Seite mit dem Submenu stehen. 

Kann es sein, dass mein Modul erst später geladen werden muss? Dieses heißt tl_page_submenu. Oder woran kann es liegen, an der nav_newscategorie vermutlich?

nav_default


```
<ul class="<?= $this->level ?> <?php if ($this->level == 'level_1'): ?>menu<?php endif; ?>" <?php if ($this->level == 'level_1'): ?>id="menu-main-menu"<?php endif; ?> role="<?= ($this->level == 'level_1') ? 'menubar' : 'menu' ?>">
  <?php foreach ($this->items as $item): ?>
    <?php if ($item['isActive']): ?>
      <li class="<?= $item['class'] ?> menu-item current-menu-item"><span class="<?= $item['class'] ?>" role="menuitem"<?php if (!empty($item['subitems'])): ?> aria-haspopup="true"<?php endif; ?>><?= $item['link'] ?></span><?= $item['customsubitems'] ?: $item['subitems'] ?></li>
<?php else: ?>
      <li<?php if ($item['class']): ?> class="<?= $item['class'] ?> menu-item <?php if ($item['customsubitems']): ?>submenu<?php endif; ?>"<?php endif; ?>><a href="<?= $item['href'] ?>" title="<?= $item['pageTitle'] ?: $item['title'] ?>"<?php if ($item['class']): ?> class="<?= $item['class'] ?>"<?php endif; ?><?php if ($item['accesskey'] != ''): ?> accesskey="<?= $item['accesskey'] ?>"<?php endif; ?><?php if ($item['tabindex']): ?> tabindex="<?= $item['tabindex'] ?>"<?php endif; ?><?php if ($item['nofollow']): ?> rel="nofollow"<?php endif; ?><?= $item['target'] ?> role="menuitem"<?php if (!empty($item['subitems'])): ?> aria-haspopup="true"<?php endif; ?>><?= $item['link'] ?></a><?= $item['customsubitems'] ?: $item['subitems'] ?></li>
    
    <?php endif; ?>
  <?php endforeach; ?>
</ul>
```

nav_newscategories für die Hauptnavigation

```
<ul class="level_2">
    <?php foreach ($this->items as $item): ?>
    <?php

// build link to jumpTo page
if( $item['jumpTo'] )
{
    $objJump = \PageModel::findByPk( $item['jumpTo'] );
    if( $objJump !== null )
        $item['href'] = $this->generateFrontendUrl( $objJump->row() );
}

    ?>
    <?php if ($item['isActive']): ?>
    <li class="<?php echo $item['class']; ?>"><span class="<?php echo $item['class']; ?>"><?php echo $item['link']; ?></span><?php if ($this->showQuantity): ?> <span class="quantity">(<?php echo $item['quantity']; ?>)</span><?php endif; ?><?php echo $item['subitems']; ?></li>
    <?php else: ?>
    <li class="<?php echo $item['class']; ?>"><a href="<?php echo $item['href']; ?>" class="<?php echo $item['class']; ?>" title="<?php echo $item['linkTitle']; ?>"><?php echo $item['link']; ?><?php if ($this->showQuantity): ?> <span class="quantity">(<?php echo $item['quantity']; ?>)</span><?php endif; ?></a><?php echo $item['subitems']; ?></li>
    <?php endif; ?>
    <?php endforeach; ?>
</ul>

```

Liebe Grüße,

Jan

**Comment by fritzmg** [comment]
> Es wird nicht das Alias der Seite, die das Modul als Submenu beinhaltet, übernommen, dadurch entsteht so ein Link http://babymanufactur.ppam.de/index/kategorie/spielen-und-schenken.html. D.h., statt „index“ sollte da das Alias der Seite mit dem Submenu stehen.

Diese URL wird von Contao generiert und das stimmt auch so - "index" kann nur weggelassen werden, wenn es keine Parameter gibt.

**Comment by jenda77** [comment]
OK, wie kriege ich es aber hin, dass da statt dessen das Richtige kommt?

**Comment by fritzmg** [comment]
Du musst im Modul die richtige Seite einstellen. Was genau ist in dem Modul nun definiert und was genau ist dann "das richtige"?

**Comment by jenda77** [comment]
Vielen Dank, das war das, was noch nicht eigestellt war. :-) Jetzt funktioniert es tadellos. Noch einmal vielen Dank und entschuldige, dass ich manchmal eingeschnappt reagierte.

**Comment by qzminski** [comment]
I see there has been quite a long discussion between two of you. Is this ticket sill relevant? Can you share (in English) what you have ended up with?

**Comment by fritzmg** [comment]
Hm, the problem was a misconfigured category list module. However, regarding the original issue, I myself can't think of a good way to introduce any such insert tag (since the target of a news category depends on the module setting).

**Comment by qzminski** [comment]
I am closing this then.

**Comment by jenda77** [comment]
Hi, I have found a solution - thanks to Fritz. :-) Best wishes, Jan

21. 12. 2015 v 16:12, Kamil Kuzminski <notifications@github.com>:

Closed #73.
